.github/workflows/stale.yml

```diff
@@ -0,0 +1,36 @@
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '0 */4 * * *'
+
+jobs:
+  prune_stale:
+    permissions:
+      issues: write  # for actions/stale to close stale issues
+      pull-requests: write  # for actions/stale to close stale PRs
+    name: Prune Stale
+    runs-on: ubuntu-latest
+    # do not run it in forked repos
+    if: github.repository == 'kubernetes-sigs/ingress2gateway'
+
+    steps:
+    - name: Prune Stale
```

Comment:
We already have k8s tooling for this, I don't think we want to maintain anything custom like on top of that

Reply:
Yep, thanks. I will remove them.

Reply:
Done